crates/construction/src/manufacturing.rs

```diff
@@ -32,41 +31,26 @@ impl Plugin for ManufacturingPlugin {
         app.add_event::<EnqueueAssemblyEvent>()
             .add_event::<ChangeDeliveryLocationEvent>()
             .add_event::<DeliverEvent>()
-            .add_system(
-                configure
-                    .in_base_set(GameSet::PostUpdate)
-                    .run_if(in_state(AppState::InGame)),
+            .add_systems(
+                PreUpdate,
+                (
+                    change_locations
+                        .run_if(in_state(GameState::Playing))
+                        .in_set(ManufacturingSet::ChangeLocations),
+                    check_spawn_locations
+                        .run_if(in_state(GameState::Playing))
+                        .before(ManufacturingSet::Produce),
+                    produce
+                        .run_if(in_state(GameState::Playing))
+                        .in_set(ManufacturingSet::Produce),
+                    deliver
+                        .run_if(in_state(GameState::Playing))
+                        .after(ManufacturingSet::ChangeLocations)
+                        .after(ManufacturingSet::Produce),
+                ),
```

Comment:
I hope you don't mind me intruding here, I'm just looking into what my PR would look like in 0.11 and had some thoughts. Feel free to tell me off if this is something I needn't worry about 😄 
Would it be a bit cleaner to have the `.run_if(in_state(GameState::Playing))` applied to entire tuples like this instead of the individual systems?

Reply:
I very much appreciate any feedback on any PR! Actually, I hope that in the future all my ORs will be reviewed by the community.

You are right that it would be cleaner. I am trying to find balance between making the diff and amount of change smaller and having the resulting code as clean as possible. Note that we can do some optional Bevy v0.11 improvements in follow up PRs.